.actor/actor.json

```diff
@@ -1,7 +1,7 @@
 {
   "actorSpecification": 1,
   "name": "docling",
-  "version": "0.0",
+  "version": "1.0",
   "environmentVariables": {},
   "dockerFile": "./Dockerfile",
   "input": "./input_schema.json",
```

Comment:
@mtrunkat do we already support `inputSchema` instead of `input` on the platform? If yes, I'd update it here too

Reply:
Yes, we do https://github.com/apify/apify-worker/commit/900996381d87d558b477eafc61fcb4539a714214#diff-d7fe5b9347f16db4864edc5c26c0b448b9df5018789afe1194417e681df9e0deR1751-R1755